napari/_qt/layer_controls/qt_labels_controls.py

```diff
@@ -163,7 +161,7 @@ def __init__(self, layer):
 
         # shuffle colormap button
         self.colormapUpdate = QtModePushButton(
-            None,
+            layer,
```

Comment:
maybe here, but I assume that it was a bug.

Reply:
also looks good